sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -36,30 +37,30 @@ protected void beforeTest() {
 
         if (interceptorManager.isPlaybackMode()) {
             client = clientSetup(credentials -> new ConfigurationClientBuilder()
-                .credential(credentials)
-                .httpClient(interceptorManager.getPlaybackClient())
-                .httpLogDetailLevel(HttpLogDetailLevel.BODY_AND_HEADERS)
-                .buildAsyncClient());
+                    .credential(credentials)
```

Comment:
We need to figure out our code style formatting as I've seen this being changed back and forth in PRs